docs/fsharp/language-reference/members/constructors.md

```diff
@@ -30,7 +30,7 @@ Initialized object that has coordinates (0, 0, 0)
 
 ## Construction of Structures
 
-Structures follow all the rules of classes. Therefore, you can have a primary constructor, and you can provide additional constructors by using `new`. However, there is one important difference between structures and classes: structures can have a default constructor (that is, one with no arguments) even if no primary constructor is defined. The default constructor initializes all the fields to the default value for that type, usually zero or its equivalent. Any constructors that you define for structures must have at least one argument so that they do not conflict with the default constructor.
+Structures follow all the rules of classes. Therefore, you can have a primary constructor, and you can provide additional constructors by using `new`. However, there is one important difference between structures and classes: structures can have a parameterless constructor (that is, one with no arguments) even if no primary constructor is defined. The parameterless constructor initializes all the fields to the default value for that type, usually zero or its equivalent. Any constructors that you define for structures must have at least one argument so that they do not conflict with the parameterless constructor.
```

Comment:
Please revert

Reply:
I've just removed this change by amending this commit

Reply:
There are still two remaining occurrences of "parameterless constructor" here, @cartermp and @NRKirby. Should those be reverted as well?

Reply:
I think these might actually be accurate.

Reply:
@rpetrusha @cartermp I will happily revert any other changes needed.

Please can you let me know if I should actually change the other 2 occurrences please?

Reply:
I think this is fine as-is; parameterless constructor is more accurate in the context of this kind of struct in F#.